djaoapp/signals.py

```diff
@@ -34,19 +35,30 @@
     providing_args=["provider", "user", "reason"])
 
 
-def _notified_recipients(organization):
+def _notified_recipients(organization, notification_slug):
     """
     Returns the organization email or the managers email if the organization
     does not have an e-mail set.
     """
-    managers = [notified.email
-        for notified in organization.with_role(saas_settings.MANAGER)]
+    def _get_optedin_users(managers, notification_slug):
+        # checking whether those users are subscribed to the notification
+        managers_ids = [manager.id for manager in managers]
+        return User.objects \
```

Comment:
Based on the implementation of `with_role` in [djaodjin-saas/saas/models.py](https://github.com/djaodjin/djaodjin-saas/blob/master/saas/models.py#L512), can't we just use the following code?

    return managers.filter(notifications__slug=notification_slug).all()

Reply:
Yes! Missed that one.